tiatoolbox/models/engine/multi_task_segmentor.py

```diff
@@ -0,0 +1,436 @@
+                (flag 0).
+        tile_output (list): A list of patch predictions, that lie within this
+            tile, to be merged and processed.
+        ref_inst_dict (dict): Dictionary contains accumulated output. The
+            expected format is {instance_id: {type: int,
+            contour: List[List[int]], centroid:List[float], box:List[int]}.
+        postproc (callable): Function to post-process the raw assembled tile.
+        merge_predictions (callable): Function to merge the `tile_output` into
+            raw tile prediction.
+        model_name (string): Name of the existing models support by tiatoolbox
+          for processing the data. Refer to [URL] for details.
+
+    Returns:
+        new_inst_dict (dict): A dictionary contain new instances to be accumulated.
+            The expected format is {instance_id: {type: int,
+            contour: List[List[int]], centroid:List[float], box:List[int]}.
+        remove_insts_in_orig (list): List of instance id within `ref_inst_dict`
+            to be removed to prevent overlapping predictions. These instances
+            are those get cutoff at the boundary due to the tiling process.
+        sem_maps (list): List of semantic segmentation maps.
+        tile_bounds (:class:`numpy.array`): Boundary of the current tile, defined as
+            (top_left_x, top_left_y, bottom_x, bottom_y).
+
+    """
+    locations, predictions = list(zip(*tile_output))
+
+    # convert from WSI space to tile space
+    tile_tl = tile_bounds[:2]
+    tile_br = tile_bounds[2:]
+    locations = [np.reshape(loc, (2, -1)) for loc in locations]
+    locations_in_tile = [loc - tile_tl[None] for loc in locations]
+    locations_in_tile = [loc.flatten() for loc in locations_in_tile]
+    locations_in_tile = np.array(locations_in_tile)
+
+    tile_shape = tile_br - tile_tl  # in width height
+
+    # as the placement output is calculated wrt highest possible resolution
+    # within input, the output will need to re-calibrate if it is at different
+    # resolution than the input
+    ioconfig = ioconfig.to_baseline()
+    fx_list = [v["resolution"] for v in ioconfig.output_resolutions]
+
+    head_raws = []
+    for idx, fx in enumerate(fx_list):
+        head_tile_shape = np.ceil(tile_shape * fx).astype(np.int32)
+        head_locations = np.ceil(locations_in_tile * fx).astype(np.int32)
+        head_predictions = [v[idx][0] for v in predictions]
+        head_raw = merge_predictions(
+            head_tile_shape[::-1],
+            head_predictions,
+            head_locations,
+        )
+        head_raws.append(head_raw)
+
+    # debate whether to check size of objects
+    if "hovernetplus" in model_name:
+        _, inst_dict, layer_map, _ = postproc(head_raws)
+        out_dicts = [inst_dict, layer_map]
+    elif "hovernet" in model_name:
+        _, inst_dict = postproc(head_raws)
+        out_dicts = [inst_dict]
+    else:
```

Comment:
@adamshephard  To cover this line it needs to support `fcn`. I have improved coverage but this needs to be addressed. If this is not required then you should remove this line.